### PR TITLE
Upload a list MC samples for Deeptau training testing

### PR DESCRIPTION
**Pull request by zhiyuanlcern**
Upload MC samples for Deeptau training testing

### CODE REVIEW

File: Production/crab/configs/DeepTau2023/Winter22Test.txt
Comment (shedprog) [inline Production/crab/configs/DeepTau2023/Winter22Test.txt:6]: I propose to organize this list a bit differently:
1) As it is HLT related datasets, and format is different from miniAOD, we could name folders with HLT_ prefix. Also by convention we use MC period, So instead of DeepTau2023 I propose HLT_Run3Winter22
2) Can this config be organized properly according to what is needed to be provided to the crab submitting script?

Comment (JordanLiKappa) [reply Production/crab/configs/DeepTau2023/Winter22Test.txt:6]: Hi, 
thanks a lot for the suggestion, for 1: have changed to the proposed name. For 2, I am not sure what is the correct structure for crab submitting, should I follow the ones in Run3Winter22?

Comment (zhiyuanlcern) [reply Production/crab/configs/DeepTau2023/Winter22Test.txt:6]: Hi,
thanks a lot for the suggestion, for 1: have changed to the proposed name. For 2, I am not sure what is the correct structure for crab submitting, should I follow the ones in Run3Winter22?

Comment (ofivite) [reply Production/crab/configs/DeepTau2023/Winter22Test.txt:6]: Yes, something like configs from `Production/crab/configs/*UL` or `Run3Winter22` folders, to make sure it is parsed correctly by `crab_submit.py`

**Review by kandrosov — CHANGES_REQUESTED** [review]
Add data and Run3Summer22D datasets

**Comment by ofivite** [comment]
And also what about `DYJetsToLL_M-50` and `WJetsToLNu` samples, I see that they were in the previous sample list but now it's only Higgs, TT, QCD and Zprime?

**Comment by zhiyuanlcern** [comment]
> 

Hi @ofivite,
The DYJets and WJets you saw before was with an older campaign (Run3Winter22) where I am switching to the latest one. I am also adding the available samples from Run3Summer22D. 
I have a question for the Data samples config file, what should be the correct format of using a specific run of the data? Right now I am using `EphemeralHLTPhysics0 /EphemeralHLTPhysics0/Run2022G-v1/RAW,run=362616` which is buggy. 
Cheers,
Jordan

File: Production/crab/configs/HLT_Run3Winter22/Data.txt
Comment (kandrosov) [inline Production/crab/configs/HLT_Run3Winter22/Data.txt:1]: use a relative path here: `Production/crab/configs/HLT_Run3Winter22/json_PU60.txt`

File: Production/crab/configs/HLT_Run3Winter22/Data.txt
Comment (kandrosov) [inline Production/crab/configs/HLT_Run3Winter22/Data.txt:4]: add all ephemeral streams 0-8

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Review by kandrosov — APPROVED** [review]
thank you @zhiyuanlcern !

**Review by ofivite — APPROVED** [review]
looks good to me!

**Comment by valeriadamante** [comment]
@zhiyuanlcern I am waiting for confirmation from TSG if it's ok to use both Run3Summer22 and Run3Summer22EE.
As soon as I get a feedback I will report here.

**Comment by valeriadamante** [comment]
> 
I checked with TSG people. 
The 'physics' ones would be Run3Summer22, available [here](https://cmsweb.cern.ch/das/request?view=list&limit=150&instance=prod%2Fglobal&input=dataset+dataset%3D%2F*%2FRun3Summer22DR*%2F*RAW*+status%3D*), with the pile up profile 2022 , in the 22EE we just asked pu Gaussian to either 70, 65,60 which maybe are also useful for some study.

**Comment by kandrosov** [comment]
thank you @valeriadamante! I think we can use both campaigns for the training to have more statistics. I think this PR can be merged, if @ofivite and @shedprog agree.

**Comment by shedprog** [comment]
> thank you @valeriadamante! I think we can use both campaigns for the training to have more statistics. I think this PR can be merged, if @ofivite and @shedprog agree.

Just one small remark: are you sure that lumiFile will be packed inside the tarball when submitting to crab, to my memory only the files in the module/submodule/{src/include/data} are copied, but I might be wrong

**Comment by kandrosov** [comment]
@shedprog lumi file is applied at the task creation stage and should not be transferred to the job site by hand.
I plan to do another PR with fixes to the code (because current code doesn't work for data) and as part of that PR I will check that `dryrun` submission work.

**Pull request merged**
